spec/fixtures/1.2_custom_nginx.template

```diff
@@ -88,6 +88,11 @@ http {
 > for i = 1, #proxy_listeners do
         listen $(proxy_listeners[i].listener);
 > end
+
```

Comment:
we should keep this file as is, since this is used to test compatibility with 1.2 version of
kong. (I'm not sure even if it's still needed, as we have passed several breaking change period).

Reply:
After this PR, we should check it, it is a legacy problem.

Reply:
let's still revert the change to this file in this PR, and revisit the necessity of it in different PR.

Reply:
done.